docker-compose.yml

```diff
@@ -14,3 +14,9 @@ services:
       POSTGRES_PASSWORD: "opensesame"
     ports:
       - "5432:5432"
+    networks:
+      - default
+
+networks:
+  default:
+    name: workspace_default
```

Comment:
Is this necessary? I think I was able to get it to work without this

Reply:
removed